packages/backend/src/telemetry/index.ts

```diff
@@ -0,0 +1,80 @@
+import { Config } from '../config/app'
+import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-grpc'
+import { Resource } from '@opentelemetry/resources'
+import {
+  MeterProvider,
+  PeriodicExportingMetricReader
+} from '@opentelemetry/sdk-metrics'
+
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-grpc'
+import { api } from '@opentelemetry/sdk-node'
+import { PgInstrumentation } from '@opentelemetry/instrumentation-pg'
+import { GraphQLInstrumentation } from '@opentelemetry/instrumentation-graphql'
+
+import { HttpInstrumentation } from '@opentelemetry/instrumentation-http'
+import {
+  BatchSpanProcessor,
+  NodeTracerProvider
+} from '@opentelemetry/sdk-trace-node'
+import { registerInstrumentations } from '@opentelemetry/instrumentation'
+import { UndiciInstrumentation } from '@opentelemetry/instrumentation-undici'
+
+// debug logger:
+// diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.DEBUG)
```

Comment:
nit: cleanup

Reply:
I was thinking about this one, but its actually quite useful to keep for seeing whether the telemetry service is working at all, particularly the connection between the backend service and the collector itself. Otherwise need to dig into docs to find this. 

might be a rare exception to keeping comments in